### ISSUE TITLE
check busy nodes and resync + start if selected when batch is submitted

### ISSUE THREAD

**Issue by qcdebug** [LogicReinc/LogicReinc.BlendFarm#60]
This looks like great easy-to-use software for self-hosted blender farms.   

I'd like to see the software retry all busy nodes periodically and sync/start rendering when they are determined to be free.  
I encountered this on my desktop earlier because blender desktop was already running a test render that I forgot about.   

This along with the sync baked files, or maybe cause a bake on the render nodes enhancement and I can see this software being used for almost any project imaginable. 

Thanks for considering this request, I understand this is free software and I'm appreciative that you want to support the community like this.

**Comment by qcdebug** [comment]
Thanks for the detailed explanation, I was thinking something simpler, though I can’t claim to understand how the queuing system works to know if it would work. 

 

Take a loop of busy nodes and wait for them to return a response of ready then do the already existing process of sync and start handing out available frames waiting to them in the process as what I would expect is normal. 

 

I have been told constantly that what I propose is never that easy and I’m not a programmer so I’d have to agree with that.  If it’s as hard as you describe then I’ll happily cancel and restart my renders as needed when I clear up my desktop from being busy.     If you do ever decide to make a studio version of this I do educational renders for people and that would absolutely be a software I’d be interested in.

**Comment by LogicReinc** [comment]
After giving it some thought, I suspect that if its constraint to queue rendering it might be a doable feature in the near-future. Though I can't give you a timeframe.

**Comment by qcdebug** [comment]
You are correct that I assumed this was useful in animation and indeed I think this would ONLY be useful on animation-related renders.    I am in no hurry with the timeline, I greatly appreciate your consideration of this request.
